### PR TITLE
Document Pod .status.nodeInfo.kubeProxyVersion deprecation

### PR DESCRIPTION
**Pull request by HirazawaUi**
KEP: https://github.com/kubernetes/enhancements/issues/4004

Implementation PR: https://github.com/kubernetes/kubernetes/pull/120954

### CODE REVIEW

**Comment by sftim** [comment]
/retitle [WIP] [placeholder]: Document Pod .status.nodeInfo.kubeProxyVersion deprecation

**Comment by dipesh-rawat** [comment]
@HirazawaUi Shouldn't we consider setting the dev-1.29 branch as the target branch instead of the main branch for this PR?

**Comment by HirazawaUi** [comment]
> @HirazawaUi Shouldn't we consider setting the dev-1.29 branch as the target branch instead of the main branch for this PR?

Okay, I've made the correction.

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 1]
```diff
@@ -220,6 +220,7 @@ For a reference to old feature gates that are removed, please refer to
 | `WinOverlay` | `false` | Alpha | 1.14 | 1.19 |
 | `WinOverlay` | `true` | Beta | 1.20 | |
 | `WindowsHostNetwork` | `true` | Alpha | 1.26 | |
+| `DisableNodeKubeProxyVersion` | `false` | Alpha | 1.29 | |
```
[the remark this answers is not in the document]
Comment (HirazawaUi) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md]: Thanks for the tip.

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 2]
```diff
@@ -760,6 +761,7 @@ Each feature gate is designed for enabling/disabling a specific feature:
 - `WinDSR`: Allows kube-proxy to create DSR loadbalancers for Windows.
 - `WinOverlay`: Allows kube-proxy to run in overlay mode for Windows.
 - `WindowsHostNetwork`: Enables support for joining Windows containers to a hosts' network namespace.
+- `DisableNodeKubeProxyVersion`: Disable setting the `kubeProxyVersion` field of the Node.
```
[the remark this answers is not in the document]
Comment (HirazawaUi) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md]: Thanks for the tip.

**Comment by dipesh-rawat** [comment]
/remove-language id
/remove-area blog
/remove-area release-eng 
/remove-sig release

_(PR now targets dev-1.29 as expected)_
/unhold

**Comment by harshitasao** [comment]
Hi @HirazawaUi  , Please take a look at [Documenting for a release - PR Ready for Review](https://kubernetes.io/docs/contribute/new-content/new-features/#pr-ready-for-review) to get your PR ready for review before **Tuesday 14th November 2023**.
Thank you!

**Comment by HirazawaUi** [comment]
> Hi @HirazawaUi , Please take a look at [Documenting for a release - PR Ready for Review](https://kubernetes.io/docs/contribute/new-content/new-features/#pr-ready-for-review) to get your PR ready for review before **Tuesday 14th November 2023**. Thank you!

According to the docs, I don't find the merge box or Ready for review, may you elaborate more?

**Comment by HirazawaUi** [comment]
@dipesh-rawat @sftim This PR is ready for review and the PR associated in k/k have been merged, would you guys mind taking a look?

**Comment by harshitasao** [comment]
Hi, @HirazawaUi ! Kubernetes v1.29 Docs shadow here. **Just a reminder that the deadline to have this PR reviewed and merged is Tuesday, 28 November**. Let me know if you need any help!

**Comment by HirazawaUi** [comment]
/assign @danwinship 
May you review the documentation for deprecate kubeProxyVersion?

**Comment by danwinship** [comment]
lgtm though I'm not sure if we need to say more?

**Pull request merged**